Dockerfile

```diff
@@ -1,14 +1,34 @@
 # Builds a docker image for Squid with config retrieved from S3
 
-FROM alpine:3.10
+FROM alpine:3.11 AS BASE
+
+RUN apk update \
+    && apk upgrade
+
+FROM BASE AS PYTHON3
+# This hack is widely applied to avoid python printing issues in docker containers.
+# See: https://github.com/Docker-Hub-frolvlad/docker-alpine-python3/pull/13
+ENV PYTHONUNBUFFERED=1
+
+RUN apk add --no-cache python3 && \
+    if [ ! -e /usr/bin/python ]; then ln -sf python3 /usr/bin/python ; fi && \
+    \
+    python3 -m ensurepip && \
+    rm -r /usr/lib/python*/ensurepip && \
+    pip3 install --trusted-host pypi.python.org --trusted-host files.pythonhosted.org --trusted-host pypi.org --no-cache --upgrade pip setuptools wheel && \
+    if [ ! -e /usr/bin/pip ]; then ln -s pip3 /usr/bin/pip ; fi
+
+FROM PYTHON3 AS SQUID
 
 ENV USER_NAME=squid
 ENV GROUP_NAME=squid
+ENV AWSCLI_VERSION="1.17.10-r0"
```

Comment:
Does this env var want to move to the AWSCLI image? Is it even needed though, as I don't see it referenced anywhere?